python/tk_multi_loader/dialog.py

```diff
@@ -1180,16 +1201,26 @@ def _add_rdo_list_view_mod(self):
             self.ui.label_2.hide()
             self.ui.thumb_scale.hide()
 
-    def _add_rdo_publish_search(self):
+    def _add_rdo_status_filter(self, hlayout):
+        '''
+        Add a combo box to sort the latest publishes by status
+        Kind of copying off the _add_rdo_status_filter
+        '''
+        filter_label = QtGui.QLabel("Filter by Status")
```

Comment:
I've since changed the code and adjusted the SizeAdjustPolicy to AdjustToContents so it shouldn't be an issue anymore